### ISSUE TITLE
Clone counter + blank default project + signed out = incorrect clone count

### ISSUE THREAD

**Issue by lisa-wolfgang** [ScratchAddons/ScratchAddons#5667]
### Description

When the "clone counter" addon is enabled and the "customizable default project" is set to the blank project preset (i.e. no sprites), any newly created project while signed out displays a clone count of -2.

### Steps to reproduce

1. Ensure that the previously described prerequisite conditions are met
2. Open [https://scratch.mit.edu/projects/editor](https://scratch.mit.edu/projects/editor)

### Environment

Browser: Firefox 108.0.2
Scratch Addons version: v1.30.1
Operating system: Ubuntu 22.10

**Comment by TylerMS887** [comment]
Weird, reproduced:
![image](https://user-images.githubusercontent.com/115214762/214710960-5d122b15-195f-42bb-a406-9372aaefb466.png)

**Comment by penguinmoose** [comment]
Also reproduced.

When I sign back in (and it auto-creates a new project), it still has a clone count of -2. After refreshing that page, the clone counter is hidden (which is normal).

**Comment by WorldLanguages** [comment]
Related: https://github.com/ScratchAddons/ScratchAddons/pull/5508#issuecomment-1367676921

> I'm not sure if we should try to correct the count. We might need to end up fixing the edge case of the edge case of the edge case. I think we could merge this, but we should avoid making additional changes like this.

**Comment by WorldLanguages** [comment]
At this point, shouldn't we count clones ourselves instead of relying on `vm.runtime._cloneCounter`?

**Comment by jeffalo** [comment]
> At this point, shouldn't we count clones ourselves instead of relying on `vm.runtime._cloneCounter`?

I think the idea was to match the way Scratch counts, because that's what the clone limit is based on.

**Comment by WorldLanguages** [comment]
> I think the idea was to match the way Scratch counts, because that's what the clone limit is based on.

Good point. When getting close to the 300 clone limit, we can't show our own reliable clone count.
